[2022-02-21 16:53:21,306] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: test_path.get_path scheduled__2022-01-28T00:00:00+00:00 [queued]>
[2022-02-21 16:53:21,319] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: test_path.get_path scheduled__2022-01-28T00:00:00+00:00 [queued]>
[2022-02-21 16:53:21,319] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-02-21 16:53:21,319] {taskinstance.py:1239} INFO - Starting attempt 1 of 1
[2022-02-21 16:53:21,319] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-02-21 16:53:21,332] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): get_path> on 2022-01-28 00:00:00+00:00
[2022-02-21 16:53:21,337] {standard_task_runner.py:52} INFO - Started process 114 to run task
[2022-02-21 16:53:21,340] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'test_path', 'get_path', 'scheduled__2022-01-28T00:00:00+00:00', '--job-id', '31', '--raw', '--subdir', 'DAGS_FOLDER/test.py', '--cfg-path', '/tmp/tmp0uooh3ft', '--error-file', '/tmp/tmpakbh4gli']
[2022-02-21 16:53:21,340] {standard_task_runner.py:77} INFO - Job 31: Subtask get_path
[2022-02-21 16:53:21,397] {logging_mixin.py:109} INFO - Running <TaskInstance: test_path.get_path scheduled__2022-01-28T00:00:00+00:00 [running]> on host 7c24e16e6467
[2022-02-21 16:53:21,449] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=test_path
AIRFLOW_CTX_TASK_ID=get_path
AIRFLOW_CTX_EXECUTION_DATE=2022-01-28T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-01-28T00:00:00+00:00
[2022-02-21 16:53:21,449] {logging_mixin.py:109} INFO - /opt/***
[2022-02-21 16:53:21,449] {python.py:175} INFO - Done. Returned value was: None
[2022-02-21 16:53:21,462] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=test_path, task_id=get_path, execution_date=20220128T000000, start_date=20220221T165321, end_date=20220221T165321
[2022-02-21 16:53:21,512] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-21 16:53:21,546] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-02-21 17:02:03,838] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: test_path.get_path scheduled__2022-01-28T00:00:00+00:00 [queued]>
[2022-02-21 17:02:03,850] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: test_path.get_path scheduled__2022-01-28T00:00:00+00:00 [queued]>
[2022-02-21 17:02:03,850] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-02-21 17:02:03,850] {taskinstance.py:1239} INFO - Starting attempt 1 of 1
[2022-02-21 17:02:03,851] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-02-21 17:02:03,869] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): get_path> on 2022-01-28 00:00:00+00:00
[2022-02-21 17:02:03,875] {standard_task_runner.py:52} INFO - Started process 112 to run task
[2022-02-21 17:02:03,877] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'test_path', 'get_path', 'scheduled__2022-01-28T00:00:00+00:00', '--job-id', '29', '--raw', '--subdir', 'DAGS_FOLDER/test.py', '--cfg-path', '/tmp/tmpcbed28u9', '--error-file', '/tmp/tmp2jos03pm']
[2022-02-21 17:02:03,878] {standard_task_runner.py:77} INFO - Job 29: Subtask get_path
[2022-02-21 17:02:03,935] {logging_mixin.py:109} INFO - Running <TaskInstance: test_path.get_path scheduled__2022-01-28T00:00:00+00:00 [running]> on host ce809d68944a
[2022-02-21 17:02:03,987] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=test_path
AIRFLOW_CTX_TASK_ID=get_path
AIRFLOW_CTX_EXECUTION_DATE=2022-01-28T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-01-28T00:00:00+00:00
[2022-02-21 17:02:03,987] {python.py:175} INFO - Done. Returned value was: /nba_database
[2022-02-21 17:02:04,006] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=test_path, task_id=get_path, execution_date=20220128T000000, start_date=20220221T170203, end_date=20220221T170204
[2022-02-21 17:02:04,050] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-21 17:02:04,096] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-02-21 19:10:47,331] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: test_path.get_path scheduled__2022-01-28T00:00:00+00:00 [queued]>
[2022-02-21 19:10:47,338] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: test_path.get_path scheduled__2022-01-28T00:00:00+00:00 [queued]>
[2022-02-21 19:10:47,338] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-02-21 19:10:47,339] {taskinstance.py:1239} INFO - Starting attempt 1 of 1
[2022-02-21 19:10:47,339] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-02-21 19:10:47,348] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): get_path> on 2022-01-28 00:00:00+00:00
[2022-02-21 19:10:47,351] {standard_task_runner.py:52} INFO - Started process 112 to run task
[2022-02-21 19:10:47,353] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'test_path', 'get_path', 'scheduled__2022-01-28T00:00:00+00:00', '--job-id', '29', '--raw', '--subdir', 'DAGS_FOLDER/test.py', '--cfg-path', '/tmp/tmpdaqt0kp9', '--error-file', '/tmp/tmp38ldf7fe']
[2022-02-21 19:10:47,353] {standard_task_runner.py:77} INFO - Job 29: Subtask get_path
[2022-02-21 19:10:47,386] {logging_mixin.py:109} INFO - Running <TaskInstance: test_path.get_path scheduled__2022-01-28T00:00:00+00:00 [running]> on host 1594888d6c38
[2022-02-21 19:10:47,418] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=test_path
AIRFLOW_CTX_TASK_ID=get_path
AIRFLOW_CTX_EXECUTION_DATE=2022-01-28T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-01-28T00:00:00+00:00
[2022-02-21 19:10:47,418] {python.py:175} INFO - Done. Returned value was: ['/home/***/.local/bin', '/usr/local/lib/python37.zip', '/usr/local/lib/python3.7', '/usr/local/lib/python3.7/lib-dynload', '/home/***/.local/lib/python3.7/site-packages', '/usr/local/lib/python3.7/site-packages', '/opt/***/dags', '/opt/***/config', '/opt/***/plugins']
[2022-02-21 19:10:47,432] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=test_path, task_id=get_path, execution_date=20220128T000000, start_date=20220221T191047, end_date=20220221T191047
[2022-02-21 19:10:47,445] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-21 19:10:47,466] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
